### DISCUSSION TITLE
Windows 11 Voice Access/Dictation

### DISCUSSION THREAD

**Discussion by jbutt9** [SafeExamBrowser/seb-win-refactoring#734] [Q&A]
Is there a way to run Voice Access for Win11 while inside the app? I've allowed it and can click it on the taskbar, but nothing happens. Have tested in all kiosk modes. I can only get it to work in debug mode and that's only if I have it running beforehand. Other dictation software seems to work fine in all modes (ReadAndWrite) - but comes with a license cost and the only feature we need is Speech to Text.

TIA

**Comment by dbuechel** [comment]
See #735.
